components/Checkboxes.tsx

```diff
@@ -29,66 +27,29 @@ export const Checkboxes: React.FunctionComponent<CheckboxesProps> = props => {
     disabled
   } = props;
 
-  return (
-    <>
-      <Fieldset className="checkboxes" legend={legend}>
-        {checkboxes.map(
-          (checkbox): JSX.Element => {
-            const labelId = `${name}-${checkbox.value.replace(
-              /\s+/g,
-              "-"
-            )}-label`;
-            const inputId = `${name}-${checkbox.value.replace(/\s+/g, "-")}`;
+  for (let i = 0; i < checkboxes.length; i++) {
```

Comment:
This should get the same treatment as radios.

Reply:
This is updated and ready for review now, thanks.